BPOR/BPOR.Rms/Models/Email/SetupCampaignViewModel.cs

```diff
@@ -20,7 +20,7 @@ public class SetupCampaignViewModel
 
     [DisplayName("Select email template")]
     public string? SelectedTemplateId { get; set; }
-    public IEnumerable<SelectListItem> EmailTemplates { get; set; } = [];
+    public TemplateList EmailTemplates { get; set; } = new ();
```

Comment:
Need to abstract this away from the Notify API. It should not be visible at this level.
We are dealing with the notification API through RMS not GOVUK Notify.